src/main/java/io/castle/client/model/AuthenticateResponse.java

```diff
@@ -0,0 +1,23 @@
+package io.castle.client.model;
+
+public class AuthenticateResponse {
```

Comment:
update according to this: https://paper.dropbox.com/doc/Server-SDK-specs-5WNJdpiGefPk2I1aAtWET#:h2=Verdict-object:

Reply:
Yes, to be fixed.